src/core/server/http/http_service.test.ts

```diff
@@ -91,7 +91,7 @@ test('creates and sets up second http server', async () => {
   const cfg = { port: 2345 };
   await serverSetup.createNewServer(cfg);
   const server = await service.start();
-  expect(server.isListening()).toBeTruthy();
+  expect(server.isListening(9222)).toBeTruthy();
```

Comment:
Where does this port number come from? I'm wary of hardcoded port numbers in tests because we often intentionally change up our ports in different testing scenarios (like if you're running multiple Kibanas), and this has bitten us before.